internal/spokes/spokes.go

```diff
@@ -114,6 +130,13 @@ type spokesReceivePack struct {
 	governor         *governor.Conn
 }
 
+func (r *spokesReceivePack) Close() error {
```

Comment:
Suggestion: The name `Close` sounds like it would be safe to call when spokes-receive-pack is finished, but in fact it should actually only be called on errors. Would it make sense to rename this to be something like `OnError` or `RemoveQuarantine`?